dashboard update notification (#1653)
vesoft-inc/nebula-docs: docs-2.0/nebula-dashboard-ent/4.cluster-operator/9.notification.md

    | Parameter     | Description                                                         |
    | -------- | ------------------------------------------------------------ |
    | Metric Type | Set a metric type. Metric type includes the node metric type and the service type (graphd,storaged,metad).              |

>     | Metric Rule | Click **+ Add condition** to set metric rules for a node or a service. It supports add composite conditions (`AND` relationship). For more information, see [Monitoring metrics](../7.monitor-parameter.md).|

Reviewer: It supports adding composite conditions (like the usage of `AND`).

>     | Metric Rule | Click **+ Add condition** to set metric rules for a node or a service. It supports add composite conditions (`AND` relationship). For more information, see [Monitoring metrics](../7.monitor-parameter.md).|

Reviewer: ```suggestion
    | Metric Rule | Click **+ Add condition** to set metric rules for a node or a service. It supports adding composite conditions (like the usage of AND). For more information, see [Monitoring metrics](../7.monitor-parameter.md).|
```
